release-notes/opensearch.release-notes-1.3.7.md

```diff
@@ -0,0 +1,12 @@
+## 2022-11-30 Version 1.3.7 Release Notes
+
+### Upgrades
+* Updated Jackson Databind to 2.13.4.2 ([#4785](https://github.com/opensearch-project/OpenSearch/pull/4785))
+* Upgrade netty to 4.1.84.Final ([#4919](https://github.com/opensearch-project/OpenSearch/pull/4919))
+* OpenJDK Update (October 2022 Patch releases) ([#5016](https://github.com/opensearch-project/OpenSearch/pull/5016))
+* Upgrade com.netflix.nebula:nebula-publishing-plugin to 4.6.0 and gradle-docker-compose-plugin to 0.14.12 ([#5316](https://github.com/opensearch-project/OpenSearch/pull/5136))
+* Updated Jackson to 2.14.0 ([#5113](https://github.com/opensearch-project/OpenSearch/pull/5113))
+* Updated Jackson to 2.14.1 ([#5356](https://github.com/opensearch-project/OpenSearch/pull/5356))
```

Comment:
Sorry, another nitpick, but there is no need to include the version 2.14.0 because it was never released. Same thing with the first line "Updated Jackson Databind to 2.13.4.2"...that has also been superseded by PR 5356.